src/css/header.css

```diff
@@ -1,3 +1,124 @@
 .header {

```

Comment:
Set `outline: transparent` for the social links and navigation links

Reply:
Done